mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PlaceAutocompleteActivity.java

```diff
@@ -67,7 +68,9 @@
     }
 
     // TODO inject
-    PlaceDetailFetcher detailFetcher = new PeliasPlaceDetailFetcher();
+    Pelias pelias = new Pelias();
```

Comment:
The `Pelias` instance passed into the `PeliasPlaceDetailFetcher` needs to be configured to use the Mapzen API key.